aten/src/ATen/native/native_functions.yaml

```diff
@@ -1280,6 +1280,9 @@
   variants: function, method
   device_check: NoCheck
   device_guard: False
+  dispatch:
```

Comment:
@cpuhrsch  when building from source with debug=0:

```
TORCH_INTERNAL_ASSERT(
          aliased_input.storage().is_alias_of(aliased_output.storage()),
            op_name);
````
The above line will be added to the codgened kernel for nested tensor because of this signature. I added a comment on the test. I really don't like this because it makes dev experience worse but I couldn't figure out a nice way around it.